<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitsClusterisationTune</class>
 <widget class="QWidget" name="SplitsClusterisationTune">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Найденные кластера</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QTableWidget" name="clustersTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>№</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>всего участков</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="maxPartsCntLabel_2">
            <property name="text">
             <string>кол-во разбиений, 2^x:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="partsCntEdit">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="minPartsCntLabel_2">
            <property name="text">
             <string>мин. кол-во участков в кластере:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="minPartsCntEdit_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="calcClustersButton">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Обновить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="clusterDistributionLabel">
            <property name="text">
             <string>распределение кластеров:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QTableWidget" name="clusterDistributionTable">
            <column>
             <property name="text">
              <string>ряд</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>распределение кластеров</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>График зависимости количества кластеров от числа разбиений</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QGraphicsView" name="clustersGraphView"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="drawClustersGraph">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Обновить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Настройка разбиений</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1">
              <widget class="QSpinBox" name="fromPartsCntEdit">
               <property name="maximum">
                <number>2048</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="fromPartsCntLabel">
               <property name="text">
                <string>от, 2^x:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="3">
              <widget class="QLabel" name="minPartsCntLabel">
               <property name="text">
                <string>Минимальное кол-во участков в кластере:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="toPartsCntLabel">
               <property name="text">
                <string>до, 2^x:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QSpinBox" name="toPartsCntEdit">
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QSpinBox" name="minPartsCntEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="4">
              <widget class="QCheckBox" name="fivePercentLinesLimitBox">
               <property name="text">
                <string>Отображать кластеры соответствующие более 5% участкам</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
